packages/components/src/Card/src/Card.module.css

```diff
@@ -0,0 +1,24 @@
+.hop-Card {
+    /* Internal Variables */
+    --border-radius: var(--hop-shape-rounded-md);
```

Comment:
we create internal css variables only for stuff that changes. the borderRadius is always the same, so it should be `--hop-Card-border-radius`